src/components/UsersList/index.ts

```diff
@@ -1,34 +1,21 @@
 import Block from "../../utils/Block";
 import template from "./usersList.hbs";
-import { UsersListItem } from "../UsersListItem";
+import { UsersListItem, UserWithRole } from "../UsersListItem";
 import { withStore } from "../../utils/Store";
 import styles from "../../styles.module.scss";
-//TODO styles
 
 export class UsersListBase extends Block {
   constructor(props: any) {
     super(props);
   }
 
   init() {
-    const usersList = this.props.users?.map(
-      (usersItem: {
-        id: number;
-        first_name: string;
-        second_name: string;
-        display_name: string;
-        login: string;
-        avatar: string;
-        email: string;
-        phone: string;
-        role: string;
-      }) => {
-        return new UsersListItem({
-          styles: styles,
-          user: usersItem,
-        });
-      },
-    );
+    const usersList = this.props.users?.map((usersItem: UserWithRole) => {
+      return new UsersListItem({
+        styles: styles,
```

Comment:
Можно чуть упростить.

```suggestion
        styles,
```

Reply:
Спасибо, исправила.